.github/workflows/params-env.yaml

```diff
@@ -6,9 +6,11 @@ on:  # yamllint disable-line rule:truthy
     paths:
       - 'manifests/base/commit.env'
       - 'manifests/base/params.env'
+      - 'manifests/base/commit-latest.env'
+      - 'manifests/base/params-latest.env'
       - 'ci/check-params-env.sh'
-      - 'ci/check-runtime-images.sh'
-      - 'manifests/base/runtime-*-imagestream.yaml'
+      # - 'ci/check-runtime-images.sh'
```

Comment:
maybe we should delete this line completely now?

Reply:
ack, i wanted too
however thought to confirm.
removing it.